core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorTableMetadata.java

```diff
@@ -30,7 +33,8 @@
     private final List<ColumnMetadata> columns;
     private final Map<String, Object> properties;
 
-    public ConnectorTableMetadata(SchemaTableName table, List<ColumnMetadata> columns)
+    @JsonCreator
+    public ConnectorTableMetadata(@JsonProperty("schemaTableName") SchemaTableName table, @JsonProperty("columns") List<ColumnMetadata> columns)
```

Comment:
Same here about columns, properties. Also it is unclear if properties are even serializable? (what values can the map contain?)

Reply:
Yes, I answer above.